cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/assembly/EntityOperatorReconciler.java

```diff
@@ -390,6 +401,24 @@ protected Future<Void> userOperatorSecret(Clock clock) {
                     .map((Void) null);
         }
     }
+    /**
+     * Manages the Entity Operator Network Policies.
+     *
+     * @return  Future which completes when the reconciliation is done
+     */
+    protected Future<Void> networkPolicy() {
+        if (isNetworkPolicyGeneration) {
+            return networkPolicyOperator
+                    .reconcile(
+                            reconciliation,
+                            reconciliation.namespace(),
+                            CruiseControlResources.networkPolicyName(reconciliation.name()),
```

Comment:
The name needs to be Entity Operator name.

Reply:
I was searching for entity operator resource file but not able to fine so that I can add the method `networkPolicyName`.

Reply:
Well, they don't have a name because they did not exist before. But you cannot just use other resources name -> that would nto work. 

But you should probably consider using the deployment names for the network policies instead of adding a new name.